climakitae/uncertain_utils.py

```diff
@@ -69,6 +70,8 @@ def _cmip_clip(self, ds):
         to_drop = [v for v in list(ds.data_vars) if v != variable]
         ds = ds.drop_vars(to_drop)
         ds = _clip_region(ds, area_subset, location)
+        if variable in ("pr"):
+            ds = _precip_flux_to_total(ds)
```

Comment:
Variable is a string, so I think it would make more sense for it to be `if variable == "pr"`

Reply:
lol, I think Eric told me to do it this way?

Reply:
@elehmer am I remembering correctly?

Reply:
Umm, I wouldn't advice to do it this way lol Nicole has it right lol

Reply:
that must have been for something else then. wild. I'll change it